colour/models/rgb/transfer_functions/tests/test_log.py

```diff
@@ -0,0 +1,175 @@
+#
+"""
+Defines unit tests for :mod:`colour.models.rgb.transfer_functions.log`
+module.
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+import unittest
+
+from colour.models.rgb.transfer_functions import (log_encoding_Log2,
+                                                  log_decoding_Log2)
+from colour.utilities import domain_range_scale, ignore_numpy_errors
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013-2020 - Colour Developers'
+__license__ = 'New BSD License - https://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-developers@colour-science.org'
+__status__ = 'Production'
+
+__all__ = [
+    'TestLogEncoding_Log2',
+    'TestLogDecoding_Log2',
```

Comment:
Extra coma at the end prevents Yapf inlining.

Reply:
Noted